N/nlminb/build_tarballs.jl

```diff
@@ -0,0 +1,68 @@
+using BinaryBuilder
+
+name = "nlminb"
+version = v"0.1.0"
+
+# Collection of sources required to build NLopt
+sources = [
+    GitSource("https://github.com/geo-julia/nlminb.f.git",
+              "6b2d460b866911e8bfca32b1217c08fb78c3111f"), # v0.1.0
+]
+
+# copied from SCALAPACK
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/nlminb.f
+cd lib && tar -xzf blas-3.8.0.tgz \
+    && cd BLAS-3.8.0 && make \
+    && cd ../.. 
+
+## compile by hand
+if [[ ${target} == *aarch64* ]] || [[ ${target} == *arm* ]]; then
+    FLAGS=""
+else
+    FLAGS="-mfpmath=sse -msse2 -mstackrealign"
+fi
+CCFLAGS="-fPIC -DNDEBUG  -Iinclude -O2 -Wall -std=gnu99 ${FLAGS}"
+FCFLAGS="-fPIC -fno-optimize-sibling-calls -O2 ${FLAGS}"
+
+gfortran ${FCFLAGS} -c src/portsrc.f -o src/portsrc.o 
+gfortran ${FCFLAGS} -c src/d1mach.f -o src/d1mach.o 
+gcc ${CCFLAGS} -c src/port2.c -o src/port.o
+gfortran -shared -static-libgcc -lm -o libnlminb.${dlext} src/portsrc.o src/d1mach.o src/port.o lib/BLAS-3.8.0/blas_LINUX.a
```

Comment:
I'm a bit confused: what's this `lib/BLAS-3.8.0/blas_LINUX.a`?  A binary blob of a BLAS library, only for Linux and presumably only for a single platform?  I didn't follow all the attempts here, what's wrong with OpenBLAS or OpenBLAS32?

Reply:
Sorry. I have no idea how to include openblas correctly. So, build manually from tgz file. 
`blas` is downloaded at the official web, http://www.netlib.org/blas/blas-3.8.0.tgz. The `lib/BLAS-3.8.0/blas_LINUX.a` is the output of openblas library. 

Thanks for your comments and review.